dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/test-applications/policyAttachmentsClient1/src/com/ibm/ws/policyattachments/client1/service1/HelloService4.java

```diff
@@ -0,0 +1,95 @@
+
+package com.ibm.ws.policyattachments.client1.service1;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+import javax.xml.ws.WebEndpoint;
+import javax.xml.ws.WebServiceClient;
+import javax.xml.ws.WebServiceException;
+import javax.xml.ws.WebServiceFeature;
+
+/**
+ * This class was generated by the JAX-WS RI.
+ * JAX-WS RI 2.2.9-b130926.1035
+ * Generated source version: 2.2
+ *
+ */
+@WebServiceClient(name = "HelloService4", targetNamespace = "http://service1.policyattachments.ws.ibm.com/",
+                  wsdlLocation = "http://localhost:8010/policyAttachmentsClient1/HelloService4?wsdl")
```

Comment:
hard-coded port # 8010?

Reply:
See the change in `dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/publish/servers/ColocatedDynamicPolicyAttachments/server.xml` where I addressed this issue.